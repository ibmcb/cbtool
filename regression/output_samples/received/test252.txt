The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_1      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |42.121.131.154  |1377115411     |none                                    |8DD20872-5363-5B99-8770-FBF86161E63B

