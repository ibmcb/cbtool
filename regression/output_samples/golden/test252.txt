 status: Sending a termination request for vm_1 (cloud-assigned uuid CBE13DA0-A167-5CB2-83C9-0E1297B06679)....
VM object A4462A1D-6DDE-5587-9C12-EACD283DD224 (named "vm_1") was sucessfully detached from this experiment.
The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |svm_stub_vmc     |uuid
|vm_3          |tinyvm              |pico32      |165.98.17.1     |simhostb1                  |SUT                 |none    |none       |none             |C4006D4B-7E44-5E9A-9DDF-547C7F05B583
|vm_6          |tinyvm              |pico32      |72.245.226.55   |simhostb3                  |SUT                 |none    |none       |none             |0A604CF0-C765-5235-AF9F-5311268DD54A
|vm_7          |tinyvm              |pico32      |203.179.21.146  |simhosta1                  |SUT                 |none    |none       |none             |D0FECFA5-F071-5981-8D33-A8547CC2F033
|vm_8          |tinyvm              |pico32      |171.50.219.241  |simhostc0                  |SUT                 |none    |none       |none             |F8B23355-7E74-5DA1-84C8-4D026E51C7C8
|vm_9          |specwebback         |platinum64  |128.46.44.147   |simhostb3                  |SUT                 |none    |none       |none             |3FA6FE92-E4A3-5785-AA58-B7E44311C66F
|vm_10         |netclient           |iron32      |152.167.48.141  |simhosta2                  |SUT                 |none    |none       |none             |18969BAC-BB5F-5955-B5C6-DA2F36C1D01B
|vm_11         |fen_hpc             |platinum64  |26.63.98.247    |simhostd1                  |LG                  |none    |none       |none             |3F519C89-970B-5C82-8F34-5ACD018DF1D8
|vm_12         |tinyvm              |iron32      |140.40.135.251  |simhostc3                  |SUT                 |none    |none       |none             |C462ACAC-7E5C-57B7-95B1-69026E3EACAE
|vm_13         |tinyvm              |iron32      |41.239.88.134   |simhostc2                  |SUT                 |none    |none       |none             |EE101EAB-A01A-509E-9C83-9C2CC129394B
|vm_14         |predictablevm       |iron32      |1.2.3.4         |simhosta3                  |SUT                 |none    |none       |none             |68C0A161-B624-5450-9F79-AEB684806C54
|vm_15         |tinyvm              |pico32      |249.122.34.127  |simhosta4                  |SUT                 |none    |none       |none             |E6D60F9C-FEDC-5CE5-BEF3-4E2DDCB8B665
|vm_16         |tinyvm              |pico32      |33.242.32.9     |simhostc2                  |SUT                 |none    |none       |none             |3A2F94B9-8EAE-523E-997E-F21F1E95CA86
|vm_17         |tinyvm              |pico32      |252.231.14.33   |simhostc3                  |SUT                 |none    |none       |none             |042C49A6-4594-5399-B3AF-43A71246D77D
|vm_18         |tinyvm              |pico32      |110.227.47.54   |simhostb2                  |SUT                 |none    |none       |none             |3FE539BD-67F8-5E57-B966-2A20F5A5D118
|vm_19         |tinyvm              |pico32      |115.33.222.1    |simhostc3                  |SUT                 |none    |none       |none             |8F8AA042-592C-5B86-8ED0-4C483D4E6762
|vm_20         |tinyvm              |pico32      |39.186.147.31   |simhostc1                  |SUT                 |none    |none       |none             |1F0731BA-18B5-5FA2-BC15-6171170E3C4A
|vm_21         |tinyvm              |pico32      |23.252.114.232  |simhosta0                  |SUT                 |none    |none       |none             |CE4F8AA5-BF02-50AB-B88B-F57C0625EF11
|vm_22         |tinyvm              |pico32      |169.1.196.104   |simhostc2                  |SUT                 |none    |none       |none             |C97E90CB-E8AF-5C2F-95E9-00DBBF4F6E64

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |1.60M                         
|Redis Version                                        |2.6.7                         
|Uptime (in seconds)                                  |459481                        
|Total Connections Received                           |2792                          
|Total Commands Processed                             |4258673                       
|Number of Keys                                       |431                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.2.2                         
|Storage Size                                         |720896                        
|Data Size                                            |143652                        
|Index Size                                           |294336                        
|Average Object Size                                  |378.031578947                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |18                            
|SVMs                                                 |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |18                            
|VMs ARRIVED                                          |22                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |4                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |114                           

 ---------------- VMC-WIDE COUNTERS ----------------
|66C80F72-DB30-5A63-B169-31C6A86D021F (simzone_c) VM RESERVATIONS|8                             
|D3E36C93-1427-5D86-BC91-CB0FA4A8E918 (simzone_d) VM RESERVATIONS|1                             
|FEBD66A7-E012-534C-A431-00CE123E50EF (simzone_a) VM RESERVATIONS|5                             
|9859398A-822B-50DC-8C37-7E3C5CBAE602 (simzone_b) VM RESERVATIONS|4                             

